### ISSUE TITLE
table alias is must be two or more leters?

### ISSUE THREAD

**Issue by juejixiaohaipi** [catfan/Medoo#988]
i use the newest version
```
$map = ['p.domain' => $this->domain, 'p.identity_tag' => $this->identity_tag, 'p.is_del' => 0];
$product = $this->register->get('db')->get('product(p)', '*', $map);
```
Fatal error: Uncaught InvalidArgumentException: Incorrect table name: product(p).
you can try this code:
```
$table = "product(p)";
// $table = "post(content)";
preg_match('/(?<table>[\p{L}_][\p{L}\p{N}@$#\-_]+)\s*\((?<alias>[\p{L}_][\p{L}\p{N}@$#\-_]+)\)/u', $table, $tableMatch);
var_dump($tableMatch);
die;

```
why not use this:
```
preg_match('/(?<table>[\p{L}\p{N}@$#\-_]+)\s*\((?<alias>[\p{L}\p{N}@$#\-_]+)\)/u', $table, $tableMatch);
```

**Comment by juejixiaohaipi** [comment]
<?php
$table = "product(p)";
// $table = "post(content)";
preg_match('/(?<table>[\p{L}_][\p{L}\p{N}@$#\-_]+)\s*\((?<alias>[\p{L}_][\p{L}\p{N}@$#\-_]+)\)/u', $table, $tableMatch);
var_dump($tableMatch);
die;

**Comment by juejixiaohaipi** [comment]
`
<?php
$table = "product(p)";
// $table = "post(content)";
preg_match('/(?<table>[\p{L}_][\p{L}\p{N}@$#\-_]+)\s*\((?<alias>[\p{L}_][\p{L}\p{N}@$#\-_]+)\)/u', $table, $tableMatch);
var_dump($tableMatch);
die;
`

**Comment by catfan** [comment]
This has been fixed on #983.

**Comment by catfan** [comment]
v2.1.1 released including this fix.
